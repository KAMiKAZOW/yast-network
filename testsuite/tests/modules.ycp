{

include "testsuite.ycp";

map READ = $[
    "probe" : $[ "system" : [], ],
];

TESTSUITE_INIT([READ], nil);
include "network/modules.ycp";

map Module = $[];

DUMP("AddModule");
TEST(``(AddModule("eth")), [], nil);
DUMP(Module);

}
