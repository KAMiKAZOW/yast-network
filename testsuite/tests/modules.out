Read	.probe.architecture nil
Read	.probe.has_apm nil
Read	.probe.has_pcmcia nil
Read	.probe.has_smp nil
Read	.probe.system []
Dump	AddModule
Return	true
Dump	$["module":"", "options":""]
